Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProvider.cpp

```diff
@@ -528,13 +528,14 @@ std::optional<MediaCapabilitiesDecodingInfo> LibWebRTCProvider::videoDecodingCap
         }
         info.powerEfficient = decodingInfo ? decodingInfo->powerEfficient : isSupportingVP9HardwareDecoder();
         info.smooth = decodingInfo ? decodingInfo->smooth : isVPSoftwareDecoderSmooth(configuration);
-    } else if (equalLettersIgnoringASCIICase(containerType, "video/h264"_s)) {
-        // FIXME: Provide more granular H.264 decoder information.
-        info.powerEfficient = info.smooth = isH264EncoderSmooth(configuration);
-    } else if (equalLettersIgnoringASCIICase(containerType, "video/h265"_s))
+    } else if (equalLettersIgnoringASCIICase(containerType, "video/h264"_s))
         info.powerEfficient = info.smooth = true;
-    else if (equalLettersIgnoringASCIICase(containerType, "video/av1"_s))
+    else if (equalLettersIgnoringASCIICase(containerType, "video/h265"_s))
+        info.powerEfficient = info.smooth = true;
+    else if (equalLettersIgnoringASCIICase(containerType, "video/av1"_s)) {
+        // FIXME: Set value to true if AV1 is only enabled when HW decoder support is enabled.
```

Comment:
I believe this is always the case, we only support av1 if we have a HW decoder available for playback
so we will return a false positive here.

Reply:
Yes, we should set `info.powerEfficient` to `true`. I'll do that as a follow-up since this is somehow orthogonal and requires test improvements.